ts/features/payments/onboarding/store/selectors/index.ts

```diff
@@ -1,13 +1,36 @@
+import * as pot from "@pagopa/ts-commons/lib/pot";
+import { createSelector } from "reselect";
+import { PaymentMethodManagementTypeEnum } from "../../../../../../definitions/pagopa/walletv3/PaymentMethodManagementType";
 import { GlobalState } from "../../../../../store/reducers/types";
+import { PaymentMethodStatusEnum } from "../../../../../../definitions/pagopa/walletv3/PaymentMethodStatus";
 
 const walletOnboardingSelector = (state: GlobalState) =>
   state.features.payments.onboarding;
 
-export const selectPaymentOnboardingRequestResult = (state: GlobalState) =>
-  walletOnboardingSelector(state).result;
+export const selectPaymentOnboardingRequestResult = createSelector(
+  walletOnboardingSelector,
+  onboarding => onboarding.result
+);
 
-export const selectPaymentOnboardingMethods = (state: GlobalState) =>
-  walletOnboardingSelector(state).paymentMethods;
+/**
+ * A payment method must be ONBOARDABLE and ENABLED in order to be onboarder
+ */
+export const selectPaymentOnboardingMethods = createSelector(
+  walletOnboardingSelector,
+  onboarding =>
+    pot.map(
+      onboarding.paymentMethods,
+      ({ paymentMethods }) =>
+        paymentMethods?.filter(
+          method =>
+            method.methodManagement ===
+              PaymentMethodManagementTypeEnum.ONBOARDABLE &&
+            method.status === PaymentMethodStatusEnum.ENABLED
+        ) || []
```

Comment:
```suggestion
        ) ?? []
```
since we have a possible undefined, I think the nullish coalescence operator is a better fit here; what do you think?

Reply:
also, I have found [this stackOverflow post](https://stackoverflow.com/questions/61480993/when-should-i-use-nullish-coalescing-vs-logical-or) about logical OR vs nullish coalescing which I found interesting, and I thought you might as well!